.travis.yml

```diff
@@ -34,13 +35,11 @@ notifications:
 
 before_script:
   - PKGS=$(go list ./...)
-  - LINT_PKGS=$(go list ./... | grep -Ev 'aws-lambda-go/lambda')
 
 script:
   - diff -u <(echo -n) <(gofmt -d ./)                              # Fail if a .go file hasn't been formatted with gofmt
   - goverage -v -covermode=atomic -coverprofile=coverage.txt $PKGS # Run all tests with coverage 
-  - go vet -v ./...                                                # static analyisis
-  - golint $LINT_PKGS                                              # lint - ignore failures for now
+  - gometalinter --vendor --disable-all --enable=vet --enable=golint ./...
```

Comment:
DEPRECATED: Use https://github.com/golangci/golangci-lint
see notes from the author of https://github.com/alecthomas/gometalinter